README.rst

```diff
@@ -43,8 +43,9 @@ If you want to run the tests, you need nose >= 1.2.1 and coverage >= 3.6.
 
 Install
 =======
+In order to perform the installation, type the following line and press "ENTER"::
```

Comment:
In order to perform the installation, run the following command from the nistats directory: